drivers/SmartThings/matter-thermostat/src/init.lua

```diff
@@ -15,10 +15,19 @@
 local capabilities = require "st.capabilities"
 local log = require "log"
 local clusters = require "st.matter.clusters"
-
+local version = require "version"
 local MatterDriver = require "st.matter.driver"
 local utils = require "st.utils"
 
+local ThermostatClusterAttributeList
+if version.api < 3 then
```

Comment:
Note that explicitly branching on `version.api` is not completely necessary; we could just require the attribute class packaged with the driver for all cases, but I did it this way to document with code the reasoning for including the cluster library file in the driver package.

Reply:
This is done.